react-scanner.config.js

```diff
@@ -0,0 +1,14 @@
+module.exports = {
+	// Crawl the entire repo
+	crawlFrom: './',
+	// Needed for properly reporting components with dot notation
+	includeSubComponents: true,
+	// Exclude usage in tests and stories.
+	globs: [ '**/!(test|stories)/!(*stories).@(js|ts)?(x)' ],
```

Comment:
There are some additional patterns you might want if you're wanting to exclude all tests.
https://github.com/Automattic/jetpack/blob/99acd1718c228485de23ebfd32fe496c9428f4c3/tools/js-tools/jest/config.base.js#L31-L33
(note the exact syntax may need to be adjusted, unfortunately different tools seem to use different libraries)